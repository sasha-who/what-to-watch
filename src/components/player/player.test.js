import React from "react";
import renderer from "react-test-renderer";
import Player from "../player/player.jsx";

it(`Player should render correctly`, () => {
  const tree = renderer
    .create(<Player />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
