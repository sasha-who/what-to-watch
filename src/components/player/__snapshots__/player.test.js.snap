// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player should render correctly 1`] = `
<div
  className="player"
>
  <video
    className="player__video"
    poster="img/player-poster.jpg"
    src="#"
  />
  <button
    className="player__exit"
    type="button"
  >
    Exit
  </button>
  <div
    className="player__controls"
  >
    <div
      className="player__controls-row"
    >
      <div
        className="player__time"
      >
        <progress
          className="player__progress"
          max={100}
          value={30}
        />
        <div
          className="player__toggler"
          style={
            Object {
              "left": "30%",
            }
          }
        >
          Toggler
        </div>
      </div>
      <div
        className="player__time-value"
      >
        1:30:29
      </div>
    </div>
    <div
      className="player__controls-row"
    >
      <button
        className="player__play"
        type="button"
      >
        <svg
          height={19}
          viewBox="0 0 19 19"
          width={19}
        >
          <use
            xlinkHref="#play-s"
          />
        </svg>
        <span>
          Play
        </span>
      </button>
      <div
        className="player__name"
      >
        Transpotting
      </div>
      <button
        className="player__full-screen"
        type="button"
      >
        <svg
          height={27}
          viewBox="0 0 27 27"
          width={27}
        >
          <use
            xlinkHref="#full-screen"
          />
        </svg>
        <span>
          Full screen
        </span>
      </button>
    </div>
  </div>
</div>
`;
